Src/FluentAssertions/Events/EventRecorder.cs

```diff
@@ -25,11 +25,13 @@ internal sealed class EventRecorder : IEventRecording, IDisposable
         /// <param name="eventRaiser">The object events are recorded from</param>
         /// <param name="eventName">The name of the event that's recorded</param>
         /// <param name="utcNow">A delegate to get the current date and time in UTC format.</param>
-        public EventRecorder(object eventRaiser, string eventName, Func<DateTime> utcNow)
+        /// <param name="threadSafeSequenceGenerator">Class used to generate a thread-safe sequence.</param>
+        public EventRecorder(object eventRaiser, string eventName, Func<DateTime> utcNow, ThreadSafeSequenceGenerator threadSafeSequenceGenerator)
```

Comment:
🙃 I would just name the `threadSafeSequenceGenerator` parameter `sequenceGenerator`.

Reply:
Done in https://github.com/fluentassertions/fluentassertions/commit/1b60c30f04754eaae986ffa081bf48cf6f44c79c